extensions/pyRevitTools.extension/pyRevit.tab/Project.panel/ptools.stack/Family.pulldown/Save Families.pushbutton/SaveFamilies_script.py

```diff
@@ -4,41 +4,83 @@
 from pyrevit import revit
 from pyrevit import forms
 from pyrevit import script
+from pyrevit.revit import FailureSwallowerContext
 
 
 logger = script.get_logger()
 output = script.get_output()
 
-
 family_dict = {}
+saved_families = 0
+count_errors = 0
+count_warnings = 0
+
+
 for family in revit.query.get_families(revit.doc, only_editable=True):
     if family.FamilyCategory:
         family_dict[
             "%s: %s" % (family.FamilyCategory.Name, family.Name)
             ] = family
+if not family_dict:
```

Comment:
I recovered `script.exit()` here and below. In other case indentation would be to long. Sorry